Gemfile

```diff
@@ -10,12 +10,13 @@ gem 'dotenv-rails'
 gem 'enum_help'
 gem 'figaro'
 gem 'foreman', require: false
-gem 'hashie'
+gem 'hashie', '~> 3.5', '>= 3.5.5'
```

Comment:
We don't need `hashie` in the Gemfile at all. It's already included in the project by omniauth.